domain/src/main/java/uk/gov/hmcts/ccd/definition/store/domain/validation/event/EventEntityCaseTypeAccessProfileValidatorImpl.java

```diff
@@ -15,26 +15,26 @@
 
 @Component
 @RequestScope
-public class EventEntityCaseTypeUserRoleValidatorImpl implements EventEntityValidator {
+public class EventEntityCaseTypeAccessProfileValidatorImpl implements EventEntityValidator {
 
     private final Map<String, ArrayList<String>> userRoleCaseTypeMap = new HashMap<>();
-    private final List<String> userRoleList = Arrays.asList("caseworker-caa", "caseworker-approver");
+    private final List<String> accessProfileList = Arrays.asList("caseworker-caa", "caseworker-approver");
```

Comment:
Rename `userRoleCaseTypeMap`.

Feel free to rename `accessProfileList` to highlight why these accessProfiles are special: e.g. `accessProfilesLimitedToOneEventPerCaseType`.

Reply:
or just add a comment to describe what this validation is trying to achieve.  As it took me a while to remember what/why it is there.

Reply:
Fixed in 6dc5c13